app/Http/Controllers/Backend/MovieController.php

```diff
@@ -0,0 +1,202 @@
+                    ' . csrf_field() . '
+                    ' . method_field('DELETE') . '
+                    <button type="submit" class="main__table-btn main__table-btn--delete" data-toggle="tooltip" title="Delete">
+                        <i class="icon ion-ios-trash"></i>
+                    </button>
+                </form>
+            </div>
+            ';
+            })
+            ->rawColumns(['id', 'name', 'slug', 'action'])
+            ->make(true);
+    }
+
+    public function nominations($id)
+    {
+        try {
+            $movie = Movie::findOrFail($id);
+            if ($movie->nominations == config('config.nomination_on')) {
+                $movie->nominations = config('config.nomination_off');
+            } else {
+                $movie->nominations = config('config.nomination_on');
+            }
+            $movie->save();
+            die($movie);
+        } catch (ModelNotFoundException $e) {
+            return $e->getMessage();
+        }
+    }
+
+    public function create()
+    {
+        $qualities = $this->getOptionValue('quality');
+        $genres = $this->getOptionValue('genre');
+        $countries = $this->getOptionValue('country');
+        $types = Type::get();
+        $tags = Tag::all();
+
+        return view('backend.movies.create')->with([
+            'qualities' => $qualities->toArray(),
+            'genres' => $genres->toArray(),
+            'countries' => $countries->toArray(),
+            'types' => $types->toArray(),
+            'tags' => $tags,
+        ]);
+    }
+
+    public function getOptionValue($name)
+    {
+        $option = Option::where('name', $name)->first();
+        $optionValues = $option->optionValues()->get();
+
+        return $optionValues;
+    }
+
+    public function store(MovieRequest $request)
+    {
+        DB::beginTransaction();
+        try {
+            $movie = new Movie();
+            $movie->description = $request->get('description');
+            $movie->name = $request->get('name');
+            $movie->name_origin = $request->get('name_origin');
+            $movie->age = $request->get('age');
+            $movie->genre = $request->get('genre');
+            $movie->runtime = $request->get('runtime');
+            $movie->release_year = $request->get('release_year');
+            $movie->quality = $request->get('quality');
+            $movie->country = $request->get('country');
+            $movie->card_cover = '';
+            $movie->user_id = 1;
+
+            if ($request->hasFile('card_cover')) {
+                $card_cover = $request->file('card_cover');
+                $name = $movie->id . '.' . $card_cover->getClientOriginalName();
+                $card_cover->storeAs('public/images', $name);
+                $movie->card_cover = '/storage/images/' . $name;
+            }
+            $movie->slug = Str::slug($request->get('name'));
+            $status = Movie::where('slug', $movie->slug)->first();
```

Comment:
```suggestion
            $movie = Movie::where('slug', $movie->slug)->first();
```
đây là movie mà, sao e lại đặt tên là slug